src/routes/rules/RulesView.vue

```diff
@@ -411,18 +388,35 @@ export default {
     this.royals = royals;
     this.oneOffs = oneOffs;
     this.sectionTitles = sectionTitles;
-  },
-  methods: {
-    goToSection(url) {
-      this.goTo(url, this.scorllOptions);
-      window.location.hash = url;
-    },
-    onIntersect(isIntersecting, entries) {
+
+    // Scrolling
+    this.scrollOptions = {
+      duration: 1000,
+      offset: -100,
+      easing: 'easeInOutCubic',
+    };
+
+    // Intersection
+    const onIntersect = (_isIntersecting, entries) => {
```

Comment:
Defining this inline inside created as it's only to be used in the context of the `intersectConfig`